salsah/src/public/js/imagebase.js

```diff
@@ -90,6 +90,10 @@ $(function() {
 			//***************
 
 
+			var sortedprops = Object.keys(resource.props).map(function(propname) {return [propname, resource.props[propname]];});
```

Comment:
@tobiasschweizer : yes, it is a bit tricky, that's why I described it at length at the beginning of the issue #557. Would like a shorter version in the code?